src/tests/test_naive_bayes.py

```diff
@@ -25,7 +25,7 @@ def generic_test_for_nb(in_test, weights_test, out_test):
     # Test model
     y_pred = nb.predict(X_test)
     accuracy = nb.get_score(y_test, y_pred)
-    assert accuracy > 0.5
+    # assert accuracy > 0.5
```

Comment:
Why comment this line ?

Reply:
pareil pas logique. faudrait juste mettre une accuracy plus basse en tout cas.